-------------------------------------------------------------------------------
Test set: com.saasovation.common.port.adapter.messaging.rabbitmq.RabbitMQPipesFiltersTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.058 sec <<< FAILURE!
testPhoneNumbersCounter(com.saasovation.common.port.adapter.messaging.rabbitmq.RabbitMQPipesFiltersTest)  Time elapsed: 2.057 sec  <<< ERROR!
com.saasovation.common.port.adapter.messaging.MessageException: Failed to create/open the queue.
	at com.saasovation.common.port.adapter.messaging.rabbitmq.BrokerChannel.<init>(BrokerChannel.java:94)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.Exchange.<init>(Exchange.java:108)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.Exchange.fanOutInstance(Exchange.java:60)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.ExchangeListener.attachToQueue(ExchangeListener.java:84)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.ExchangeListener.<init>(ExchangeListener.java:38)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.RabbitMQPipesFiltersTest$PhoneNumberExecutive.<init>(RabbitMQPipesFiltersTest.java:156)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.RabbitMQPipesFiltersTest.setUp(RabbitMQPipesFiltersTest.java:67)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:606)
	at com.rabbitmq.client.ConnectionFactory.createFrameHandler(ConnectionFactory.java:445)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:504)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:533)
	at com.saasovation.common.port.adapter.messaging.rabbitmq.BrokerChannel.<init>(BrokerChannel.java:89)
	... 26 more

